readme.md

```diff
@@ -253,6 +254,11 @@ Made with Electron.
 - [Architecture overview](https://github.com/ilyavorobiev/atom-docs/blob/master/atom-shell/Architecture.md)
 
 
+## Books
+
+- [Cross Platform Desktop Applications](https://www.manning.com/books/cross-platform-desktop-applications)
```

Comment:
Hi, 

The book's page has been updated on manning.com, and the book is currently #6 in this week's top selling list on their site.